rescue/benches/rescue.rs

```diff
@@ -0,0 +1,40 @@
+use std::any::type_name;
+
+use criterion::{criterion_group, criterion_main, BenchmarkId, Criterion};
+use p3_field::PrimeField64;
+use p3_mds::mersenne31::MdsMatrixMersenne31;
+use p3_mds::MdsPermutation;
+use p3_mersenne_31::Mersenne31;
+use p3_rescue::{BasicInverseSboxLayer, InverseSboxLayer, Rescue};
+use p3_symmetric::permutation::CryptographicPermutation;
+use rand::distributions::{Distribution, Standard};
+use rand::{thread_rng, Rng};
+
+fn bench_rescue(c: &mut Criterion) {
+    rescue::<Mersenne31, MdsMatrixMersenne31, BasicInverseSboxLayer, 12, 5>(c);
+}
+
+fn rescue<F, Mds, Isl, const WIDTH: usize, const ALPHA: u64>(c: &mut Criterion)
+where
+    F: PrimeField64,
+    Standard: Distribution<F>,
+    Mds: MdsPermutation<F, WIDTH> + Default,
+    Isl: InverseSboxLayer<F, WIDTH, ALPHA> + Default,
+{
+    // TODO: Should be field specific?
+    const NUM_ROUNDS: usize = 7;
```

Comment:
Is it with 40% security margin? I believe RP minimal number of rounds is 8 based on the specs.

Reply:
Good point, I was thinking of RPO but I guess this isn't quite RPO.